homeassistant/components/geniushub/sensor.py

```diff
@@ -58,6 +58,32 @@ def name(self):
         """Return the name of the sensor."""
         return self._name
 
+    @property
+    def icon(self):
+        """Return the icon of the sensor."""
+        child_values = self._device._info_raw['childValues']  # noqa; pylint: disable=protected-access
+
+        last_comms = datetime.fromtimestamp(child_values['lastComms']['val'])
```

Comment:
You want to use `utc_from_timestamp` from our `util.dt` package.

Reply:
yes, a silly mistake - switched to UTC now.